src/Proj4.jl

```diff
@@ -1,5 +1,8 @@
 module Proj4
 
+using BinDeps
+@BinDeps.load_dependencies
+
 const libproj = "libproj"
```

Comment:
Yes, I don't yet know the details of how `BinDeps` relates to precompilation.  The current setup with the string constant for `libproj` seems clearly ok for precompilation, whereas caching a pointer to a `dlopen()`'d libproj would clearly be bad ;-)  I don't know if BinDeps does something sane here yet or not...